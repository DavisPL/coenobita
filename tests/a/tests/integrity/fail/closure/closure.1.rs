#![allow(unused_variables)]
#![allow(unused_assignments)]

fn main() {
    #[cnbt::tag({*}{*} fn({a}{a}) -> {b}{b})]
    let f = |x: i32| x;
}
