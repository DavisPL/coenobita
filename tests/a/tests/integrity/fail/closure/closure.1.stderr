error: expected {b}{b}, found {a}{a}
 --> tests/integrity/fail/closure/closure.1.rs:6:22
  |
6 |     let f = |x: i32| x;
  |                      ^
